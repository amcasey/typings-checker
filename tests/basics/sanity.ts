// (this file has no errors and no type/error assertions)
function foo() {
  return true;
}
