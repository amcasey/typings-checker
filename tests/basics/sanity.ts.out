tests/basics/sanity.ts: 0 / 0 checks passed.
